### ISSUE TITLE
[Task]: implement support for cinder's create report webhook

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons#14975]
### Description

part of https://mozilla-hub.atlassian.net/browse/AMOENG-961 where we are changing how escalations in cinder work. There are cinder workflows that process escalations to legal (and AMO) - they work by creating a new report, under a new job, in an escalation queue when the job is resolved with a specific policy/action.  To be notified when this happens we need to monitor the webhook requests for this type of event too.

As currently the webhook payload for the create report does not contain a reference to the job that triggered the workflow - only the decision id of the decision on that job - we have 3 options.
- capture the decision with the create decision webhook, so we can cross-reference the decision id in the `create_report` webhook.  Simultaneous requests are a problem, which can only really be overcome by delaying processing the job or other hacks.
- make an API call back to cinder to get more information on the decision.
- Wait for a [feature-request](https://lindie.app/share/6a21d831b39351d7c6fe898f6d22619af62dde98) that has been made to the Cinder team, to include the job id of the job that triggered the workflow in the payload.  This is the cleanest solution and - unless the lead time turns out to be unacceptable - what we'll go with.

### Acceptance Criteria

```[tasklist]
  ### Milestones/checkpoints
  - [ ] A workflow creating a new job on the back of a decision (e.g. Escalate to Legal) triggers an update of the job id to AMO.
```

### Checks

- [X] If the issue is ready to work on, I have removed the "needs:info" label.
- [X] If I have identified that the work is specific to a repository, I have removed "repository:addons-server" or "repository:addons-frontend"

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMOENG-996)

**Comment by diox** [comment]
Blocked on Cinder implementing the [feature request linked above](https://lindie.app/share/6a21d831b39351d7c6fe898f6d22619af62dde98).

**Comment by eviljeff** [comment]
The Cinder developers have gone a different way, after we explained our use-case of using workflows to allow escalations to multiple queues -  they are going to expand the queue setting to allow multiple escalation queues to be defined, so we won't need to monitor for new reports with this endpoint.

If we want to reimplement escalations to AMO using that new feature we'll need to monitor the job action webhook instead but we can open a new issue for that as a feature improvement later.
